spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/ThreadPoolSpec.scala

```diff
@@ -121,4 +122,49 @@ class ThreadPoolSpec extends FlatSpec with Matchers {
       results.foreach(_.get())
     }
   }
+
+  "setFlushDenormalState" should "influence float arithmetic operations" in {
+    val threadPool = new ThreadPool(1)
+
+    threadPool.invokeAndWait2( (0 until 1).map(i => () => {
```

Comment:
Maybe your check should be with `mkldnn` engine type.

Reply:
Since it is a test on floating point computing, the engineType should not have an effect on the result?